main.py

```diff
@@ -1,111 +1,27 @@
-    "yummy", "admirable", "brilliant", "delighted", "effortless", "festive", "genuine",
-    "helpful", "innovative", "joyful", "kindness", "luminous", "magnificent", "noble",
-    "outstanding", "playful", "radiant", "sublime", "thrilling", "unforgettable",
-    "victorious", "whimsical", "zesty"
-]
+from constants import random_words, random_symbols
 
-random_symbols = [
-    "!", "@", "#", "$", "%", "^", "&", "*", "(", ")", "-", "_", "=", "+", 
-    "[", "]", "{", "}", "|", "\\", ":", ";", "'", "\"", "<", ">", ",", ".", 
-    "?", "/", "~", "`"
-]
+def random_year():
+    return random.randint(1100, 2099)
 
-years = [
-    1900, 1901, 1902, 1903, 1904, 1905, 1906, 1907, 1908, 1909,
-    1910, 1911, 1912, 1913, 1914, 1915, 1916, 1917, 1918, 1919,
-    1920, 1921, 1922, 1923, 1924, 1925, 1926, 1927, 1928, 1929,
-    1930, 1931, 1932, 1933, 1934, 1935, 1936, 1937, 1938, 1939,
-    1940, 1941, 1942, 1943, 1944, 1945, 1946, 1947, 1948, 1949,
-    1950, 1951, 1952, 1953, 1954, 1955, 1956, 1957, 1958, 1959,
-    1960, 1961, 1962, 1963, 1964, 1965, 1966, 1967, 1968, 1969,
-    1970, 1971, 1972, 1973, 1974, 1975, 1976, 1977, 1978, 1979,
-    1980, 1981, 1982, 1983, 1984, 1985, 1986, 1987, 1988, 1989,
-    1990, 1991, 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
-    2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009,
-    2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018, 2019,
-    2020, 2021, 2022, 2023, 2024, 2025, 2026, 2027, 2028, 2029,
-    2030, 2031, 2032, 2033, 2034, 2035, 2036, 2037, 2038, 2039,
-    2040, 2041, 2042, 2043, 2044, 2045, 2046, 2047, 2048, 2049,
-    2050
-]
-
-def random_generator():
-    number = random.uniform(0,1)
-    if number >= 0.5:
-        return True
-    elif number <= 0.5:
-        return False
 
 def pass_generator(custom_word, words_amount):
-    
     password = ""
-    words_to_use = []
-    words_to_use.append(custom_word)
-    
-    for i in range(words_amount-1):
-        words_to_use.append(random.choice(random_words))
-
-    boolean = random_generator()
+    random_words_to_use = random.sample(random_words, words_amount-1)
+    words_to_use = [custom_word] + random_words_to_use
+    capitalised_word_pos = random.randint(0, words_amount-1)        
+    random.shuffle(words_to_use)
+
+    for idx, word in enumerate(words_to_use):
+        current_word = word
+        if idx == capitalised_word_pos:
+            current_word = word.capitalize() + random.choice(random_symbols)
+        password += current_word
     
-    while(len(words_to_use) != 0):
-        word = random.choice(words_to_use)
-        words_to_use.remove(word)
-        #print(boolean)
-        if boolean:
-            word = word.capitalize() + random.choice(random_symbols)
-        password += word
-        
-        boolean = not boolean
-        
-
-    password += str(random.choice(years))
-
+    password += str(random_year())
     return password
 
-#password1 = pass_generator("firstTry", 3)
```

Comment:
renamed to `test_password` as `password1` naming convention seems a bit odd in python (maybe not so odd in Java).